### DISCUSSION TITLE
text-decoration property

### DISCUSSION THREAD

**Discussion by djayMehrotra** [microsoft/griffel#192] [Q&A] [answered]
how to use text decoration property?

**Comment by layershifter** [comment, marked as answer]
FYI: I created an issue to create a helper, #193.

Until the helper function is there - you need to use CSS longhands directly, for example:

```ts
makeStyles({
  root: {
    // instead textDecoration: 'underline',
    textDecorationLine: 'underline',
  }
})
```

You can use Chrome/Devtools as a guide, for example:

![image](https://user-images.githubusercontent.com/14183168/184328921-04e8eb98-aafd-48c3-9a2a-3d8e0f050a37.png)

**Reply by djayMehrotra** [reply]
Thanks @layershifter for your help
